ntpsync.yaml

```diff
@@ -0,0 +1,87 @@
+#
+# Copyright 2022 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Number of attempts to contact a ntp server before giving up.
+request_retries: 3
+# Number of randomly selected ntp servers to interrogate.
+num_servers: 4
+# Number of servers who must agree with local time.
+server_threshold: 3
+# Maximum number of seconds the local time is allowed to drift from the
+# response of a ntp server
+max_time_delta: 1
+# Period (in seconds) for polling ntp servers
+period: 60
+# List of servers to contact. Many DNS names resolves to multiple A records.
+servers:
+  #
+  # stratum 1 servers:
+  #
+  # Apple AS6185
+  - "time.apple.com"
+
+  # Google AS15169
+  - "time.google.com"
+  - "time1.google.com"
+  - "time2.google.com"
+  - "time3.google.com"
+  - "time4.google.com"
+
+  # Facebook AS32934
```

Comment:
I'm gonna be nitpicky, can we remove fb from the list? You can ask offline why :)